docs/settings/settings-xkb.asciidoc

```diff
@@ -12,6 +12,7 @@ For more {kib} configuration settings, see <<settings>>.
 
 include::alert-action-settings.asciidoc[]
 include::apm-settings.asciidoc[]
+include::banners-settings.asciidoc[]
```

Comment:
Your file isn't showing up in the side navigation.  The correct file to include `banners-settings.asciidoc` in is docs/setup/settings.asciidoc. I don't know why we have 2 such similar files.  Looks like we have some cleaning up to do to avoid future confusion.

Reply:
Thanks, I missed that one.